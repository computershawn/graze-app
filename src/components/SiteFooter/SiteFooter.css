footer {
  text-align: center;
  position: fixed;
  left: 0;
  bottom: 0;
  width: 100%;  
  background-color: #fafafa;
  padding: 10px 0;
}

.app-title-footer {
  font-style: italic;
}