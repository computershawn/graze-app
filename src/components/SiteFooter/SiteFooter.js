import React, { Component } from 'react';
import './SiteFooter.css';

class SiteFooter extends Component {
  copyrightYear() {
    let d = new Date()
    return d.getFullYear()
  }
  
  render() {
    return (
    <footer>&copy; <span className="app-title-footer">graze</span> {this.copyrightYear()}</footer>
    );
  }
}


export default SiteFooter;