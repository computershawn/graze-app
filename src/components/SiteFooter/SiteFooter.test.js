import React from 'react'
import ReactDOM from 'react-dom'
import { BrowserRouter } from 'react-router-dom'
import SiteFooter from './SiteFooter'
import { shallow } from 'enzyme'
import toJson from 'enzyme-to-json'


it('renders without crashing', () => {
  const div = document.createElement('div');
  ReactDOM.render(
    <BrowserRouter>
      <SiteFooter />
    </BrowserRouter>,
    div
  );
})

it('renders empty given no tabs', () => {
  const wrapper = shallow(<SiteFooter />)
  expect(toJson(wrapper)).toMatchSnapshot()
})