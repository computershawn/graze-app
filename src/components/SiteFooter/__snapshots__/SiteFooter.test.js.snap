// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders empty given no tabs 1`] = `
<footer>
  Â© 
  <span
    className="app-title-footer"
  >
    graze
  </span>
   
  2019
</footer>
`;
